### PR TITLE
Analysis/ phylocanvas react functional component

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Updated phylocanvascomponent to a functional react component. Updated js/apis/* code to use setBaseUrl.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
~* [ ] Tests added (or description of how to test) for any new features.~
~* [ ] User documentation updated for UI or technical changes.~

### CODE REVIEW

File: src/main/webapp/resources/js/apis/linker/linker.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import axios from "axios";
+import { setBaseUrl } from "../../utilities/url-utilities";
 
-const BASE_URL = `ajax/ngs-linker`;
+const BASE_URL = setBaseUrl(`ajax/ngs-linker`);
```
Comment (joshsadam) [inline src/main/webapp/resources/js/apis/linker/linker.js]: Were you having problems with these urls?  Any ajax request gets intercepted before being sent out and updated with the base url.  See line 76 in app.js

Comment (deepsidhu85) [reply src/main/webapp/resources/js/apis/linker/linker.js]: These ones were not causing any issues but to keep it consistent with the other files I updated these to match those files

Comment (joshsadam) [reply src/main/webapp/resources/js/apis/linker/linker.js]: Ok, I was worried my interceptor was failing, thanks. :+1:

Comment (deepsidhu85) [reply src/main/webapp/resources/js/apis/linker/linker.js]: Oh yeah no worries about that. It works great!

**Pull request merged**